pkg/api/v1/schema.go

```diff
@@ -76,9 +76,17 @@ type CPU struct {
 }
 
 type Firmware struct {
```

Comment:
Can we rename the `Firmware` field to `BIOS`? A general BIOS section would allow to also specify things like uefi,... under that section.

Reply:
General `Firmware` section allows exactly that, while making the distinction between BIOS and UEFI clear. For that reason, I'd prefer staying with `Firmware`.